docs/montecarlo/sourceintegration.rst

```diff
@@ -0,0 +1,102 @@
+********************************
+Direct source integration method
+********************************
+
+.. note::
+
+  The following is provisional information, describing a code path that is right now active in the downbranch scheme.
+
+
+One way to increase the speed of the monte carlo procedure is by improving final method of generating the spectra so that the quality of spectra produced by a given amount of packets is increased. This is the goal of the source integration scheme of :cite `Lucy99b`, which replaces the simple binning of the escaping packets with a method based on the formal integral of the emergent intensity.
+
```

Comment:
As discussed previously, I would say you are not increasing the speed of the Monte Carlo procedure (you are not interfering with the MC procedure itself), but you are significantly increasing the S/N in the resulting spectra. Do not just say "quality of spectra". Be specific and say the method increases the signal to noise ratio.